script/Ciel/Day 13/13_01_01_CIEL13_1.txt

```diff
@@ -116,37 +116,38 @@ It's repulsive.
 [sha:72e2e9111fbdb75ee545d581c5e8246ba849dc45]{
 -- Page 7, Offset 33089.
 -- 終わりが定められていながら、限界が見えていながら、際限なく増え続け生き続ける意味が分からない。
-As my end is already decided, and as my own limits are in sight, I find myself no longer understanding the point of endlessly living on.
+My end has already been ordained, and my own limits are in sight. As such, I no longer see the purpose in continuing to live on eternally.
 }
 [sha:3f64ad09344cc23efeb1a83abc58635f1839a4d5]{
 -- Page 7, Offset 33090.
 -- 不始末すぎて無様すぎる。見えているモノが全て不気味に見えてしまう。
-I'm so incomplete that I'm hideous. Everything that I see with my eyes feels uncanny.
+I'm so incomplete that it's pathetic. Everything reflected in my eyes looks ominous.
 }
 [sha:8e01ff965d0ead9fcdb92ca4e3daff9182d1b6a2]{
 -- Page 8, Offset 33091.
 -- かといって目を閉じれば、思い出すのは人を殺した時の感触だけだ。
-Having said that, when I close my eyes, all I remember are the feelings of every time I've killed a person.
+Despite that, when I close my eyes, all I can remember is the sensation of murder.
 }
 [sha:10ecd2e0f5e50e00b6bdde00bf076a4f814384d0]{
 -- Page 8, Offset 33092.
 -- 硬い、血のかよっていない刃物が、
-The feeling of a cruel, hard blade,
+The feeling of a cruel, hard blade―――
 }
 [sha:cf3b5ed11e18b2a9f9fb48d1bb50fc58be101717]{
 -- Page 8, Offset 33093.
 -- 柔らかな、脈打つ肉と一体化していく感触。
-becoming one with the tender, pounding flesh.
+becoming one with the tender, pulsating flesh.
 }
 [sha:7912f3589446c595e0fac751242aa85453d158ec]{
 -- Page 8, Offset 33094.
 -- 知性持つ高等動物にとって障害になるほどの精神損傷。舌を噛み切りたくなるほどのおぞましい感触。もうこれ以上はないのだと絶望するほどの行き止まり。アレを知ってしまえば元の位置になど戻れない。
-For an higher intelligent being, it is mentally damaging to the point of disability. A repulsive feeling to the point of making you want to bite your own tongue off. A dead end causing despair, knowing that there's nothing else. Once you understand that, there's no turning back.
+To a being of higher intelligence, the mental strain is so severe that it can cause disability. A sensation so repulsive that it makes you want to bite your tongue off. A dead end that only leads to an unparalled despair. There's no turning back after experiencing that once.
 }
 [sha:38b29f8876aba4580217285d7ca8cedff9d85026]{
 -- Page 9, Offset 33095.
 -- とりわけ始末の悪いことに、俺の場合はきっかけの相手が最悪なまでに最高だった。
-To make matters worse, the one who triggered it all is the worst one you could possibly find.
+// not sure wtf 最悪なまでに最高 means
```

Comment:
i mean breaking it down, i think you've got the right TL lit. "as far as being the worst is concerned, the best"

plus, this TL is a great shoutout to my boys down at redlettermedia in milwaulkee